wcfsetup/install/files/lib/system/form/builder/field/AbstractFormFieldDecorator.class.php

```diff
@@ -0,0 +1,523 @@
+<?php
+
+namespace wcf\system\form\builder\field;
+
+use wcf\data\IStorableObject;
+use wcf\system\form\builder\field\dependency\IFormFieldDependency;
+use wcf\system\form\builder\field\validation\IFormFieldValidationError;
+use wcf\system\form\builder\field\validation\IFormFieldValidator;
+use wcf\system\form\builder\IFormParentNode;
+
+/**
+ * Decorator for form field objects.
+ *
+ * @author  Peter Lohse
+ * @copyright   2001-2021 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package WoltLabSuite\Core\System\Form\Builder\Field
+ * @since   5.4
+ */
+abstract class AbstractFormFieldDecorator implements IFormField
+{
+    /**
+     * decorated field
+     *
+     * @var IFormField
+     */
+    protected $field;
+
+    /**
+     * Creates a decorator object for form fields
```

Comment:
This is pretty tautological as well. I can't really think of a better comment either. Maybe just completely remove the PHPDoc for `__construct()`.